x-pack/test/api_integration/apis/cloud_security_posture/rules/v1.ts

```diff
@@ -0,0 +1,237 @@
+/*
```

Comment:
all test files should be have a suffix `test.ts`

Reply:
i thought thats only for Jest test ? this one is FTR no ?

Reply:
MB, missed the file name